pkgs/top-level/all-packages.nix

```
@@ -31288,6 +31288,9 @@ with pkgs;
 
   denaro = callPackage ../applications/finance/denaro { };
 
+  devpod = (callPackage ../development/tools/devpod { }).cli;
+  devpod-desktop = (callPackage ../development/tools/devpod { }).desktop;
```

```suggestion
  inherit (callPackage ../development/tools/devpod { }) devpod devpod-desktop;
```

`cli` needs a rename for a this to work